bin/sensei.js

```diff
@@ -195,21 +191,26 @@ cli(
       "build the static web site",
       describePositionalArguments
     )
-    .config(
-      fs.existsSync(".sensei.json")
-        ? JSON.parse(fs.readFileSync(".sensei.json", "utf-8"))
-        : {}
-    )
+    .option("config", {
+      type: "string",
+      describe:
+        "Path to JSON config file. If not absolute path, then relative to 'material' directory.\n\
+        Config file is a simple JSON to define CLI options, e.g '{\"slideWidth\": 1024, \"slideHeight\": 768}'",
+      default: ".sensei.json",
+    })
     .option("material", {
       type: "string",
       describe:
         "Path to the folder containing the training material. Defaults to the current working directory.",
       default: ".",
+      coerce(arg) {
+        return path.resolve(arg);
+      },
     })
     .option("slug", {
       type: "string",
       describe:
-        "Training name used in PDF files and HTML page titles. Defaults to the name of the current working directory.",
+        "Training name used in PDF files and HTML page titles. Defaults to the name of the 'material' directory.",
```

Comment:
Maybe not clear, "Defaults to the name of the directory of the 'material' option."? (sounds weird)

Reply:
I think the current wording is fine :)